llpc/tool/llpcCompilationUtils.cpp

```diff
@@ -445,22 +441,20 @@ Error processInputPipeline(ICompiler *compiler, CompileInfo &compileInfo, const
 // @returns : `ShaderModuleData` on success, `ResultError` on failure
 static Expected<ShaderModuleData> processInputSpirvStage(const InputSpec &spirvInput, bool validateSpirv) {
   assert(isSpirvBinaryFile(spirvInput.filename) || isSpirvTextFile(spirvInput.filename));
-  std::string spvBinFile;
+  BinaryData spvBin;
```

Comment:
Just a nit. Prefer using `spvBin = {}` for initialization.

Reply:
Done